addons/sms/wizard/sms_composer.py

```diff
@@ -65,7 +65,7 @@ def default_get(self, fields):
     template_id = fields.Many2one('sms.template', string='Use Template', domain="[('model', '=', res_model)]")
     body = fields.Text(
         'Message', compute='_compute_body',
-        precompute=True, readonly=False, store=True, required=True)
```

Comment:
I think this field should be required, it doesn't make sense to send an empty SMS

Reply:
@abdounasser202 thanks for the feedback, updated